libs/server/Resp/AdminCommands.cs

```diff
@@ -270,6 +270,75 @@ private bool ProcessAdminCommands<TGarnetApi>(RespCommand command, ReadOnlySpan<
                     errorCmd = "ping";
                 }
             }
+            else if (command == RespCommand.HELLO)
```

Comment:
General comment: probably good to add a test for this command

Reply:
Added a test